directory/cli/src/user.py

```diff
@@ -74,7 +74,93 @@ def show(login):
             error = '{}: user not found'.format(login)
             raise click.ClickException(error)
 
-    wrapper_commands = [
+    if not utils.advanced_mode_enabled():
+        @user.command(help='Create a new user')
+        def create():
+            click.echo('Please enter details for the following:')
+
+            params = {
+                        'login': click.prompt('  Username'),
+                        'first': click.prompt('  First name'),
+                        'last': click.prompt('  Surname'),
+                        'email': click.prompt('  Email'),
+            }
+            if click.confirm('Would you like to add a UID to this user?'):
+                params = { **params, 'uid': click.prompt('  UID') }
+
+            group_id = _get_group_id('clusterusers')
+
+            if group_id:
+                params = { **params, 'gidnumber': group_id[0] }
+
+            wrapper = ipa_wrapper_command.create_ipa_wrapper(
+                'user-add',
+                argument_name='login',
+                transform_options_callback=_transform_create_options,
+                handle_result_callback=_handle_create_result,
+            )
+            wrapper(**params)
+
+        @user.command(help='Modify an existing user')
+        def modify():
+            click.echo('Please enter the name of the user you want to modify:')
+            user = click.prompt('  Username')
+            user_find_args = ['--login={}'.format(user)]
+            try:
+                user_data = list_command.find_data(
+                        'user-find',
+                        user_find_args,
+                        all_fields=True
+                )[0]
+            except IpaRunError:
+                # Matches error shown for user show, can extract logic to a method
+                # in the future
+                error = '{}: user not found'.format(user)
+                raise click.ClickException(error)
+
+            click.echo('Modifying user: %s' % user)
```

Comment:
Do we need to output the username of the IPA user being modified, since the Userware user will have just entered this immediately before?

Reply:
Not necessary no, I was torn on this. But opted to add it as a confirmation of who you are modifying, can be very easily removed though.

Reply:
OK sure, that's a reasonable decision :slightly_smiling_face: - since we're one slightly in favour and one slightly against I'll leave it up to @ColonelPanicks to be the tie breaker, and it's not particularly important to decide this now (or before this PR is merged).

Reply:
Yep we currently just accept the current username to modify, and don't provide the option of giving a new one to overwrite it with. Can just delete this line then @atoghill.